t/22-dashboard.t

```diff
@@ -23,7 +23,7 @@ use Mojo::Base -signatures;
 use Test::Mojo;
 use Test::Warnings ':report_warnings';
 use OpenQA::Test::Case;
-use OpenQA::Test::TimeLimit '10';
+use OpenQA::Test::TimeLimit '50';
```

Comment:
This is a high increase. Is this really required?

Reply:
I know, please this is still WIP